pkg/config/config_test.go

```diff
@@ -43,72 +41,6 @@ var _ = Describe("Config", func() {
 			gomega.Expect(err).To(gomega.BeNil())
 		})
 
-		It("should fail on wrong DefaultUlimits", func() {
```

Comment:
Should all of this have moved into a test file that only builds on local?

Reply:
Yup I forgot to add new test files.  Fixed.